### ISSUE TITLE
AttributeError: module 'coloredlogs' has no attribute 'CAN_USE_BOLD_FONT'

### ISSUE THREAD

**Issue by AbdulRahmanAlHamali** [xolox/python-coloredlogs#82]
Hello, thank you for this beautiful package :)  

In version 14.0 we are facing this error:
```
AttributeError: module 'coloredlogs' has no attribute 'CAN_USE_BOLD_FONT'
```
Was this removed or something?

**Comment by roedoejet** [comment]
Yes, looks like it was removed here: 044402534c2dd02ff340df2950e485ce4999386c was there a reason for the removal? This is a breaking change in our code.

**Comment by xolox** [comment]
Hi both, thanks for the feedback and sorry for the breaking change. A couple of thoughts on this:

- The `CAN_USE_BOLD_FONT` variable was never intended to be used outside of `coloredlogs`.
- The fact that it was documented can be taken to imply that it was part of the public API, however to be honest I didn't put that much thought into it - I simply tend to document all code I write.
- Long after the first releases of `coloredlogs` were published, I picked up the habit of defining `__all__` in the Python packages I publish, to avoid this ambiguity. Unfortunately it takes time to go back and retroactively add this to all my projects (I hadn't gotten to `coloredlogs` yet).
- Release 14.0 bumped the major version number because it contains these kinds of backwards incompatible changes (this is what major version numbers are for). If this is -understandably- too big a risk, pin the major version number (this is what requirements pinning is for).

About why `CAN_USE_BOLD_FONT` was removed:

- Previously `colorama` was the only way to get colors on Windows.
- Starting with coloredlogs 14.0 there's a (better) alternative that doesn't have the same quirks.
- Compensating for quirks in `colorama` was basically dealing with a "leaky abstraction" (edge case) and like most developers I'd rather have less than more edge cases in my code, so with the advent of Windows 10 native ANSI support I decided to remove the special handling of `colorama`.
